perfkitbenchmarker/pkb.py

```diff
@@ -320,9 +321,17 @@ def RunBenchmarks(publish=True):
 
   if FLAGS.run_uri is None:
     if FLAGS.run_stage not in [STAGE_ALL, STAGE_PREPARE]:
-      logging.error(
-          'Cannot run "%s" with unspecified run_uri.', FLAGS.run_stage)
-      return 1
+      # Attempt to get the last modified run directory.
+      run_uri = vm_util.GetLastRunUri()
+      if run_uri:
+        FLAGS.run_uri = run_uri
```

Comment:
should you make sure the user has not already set this?  Seems like you might be overriding the user here.